model.py

```diff
@@ -35,20 +35,25 @@ def __init__(self, num_const, num_var, num_hidden, num_out, num_groups, num_subm
 
         self.submodels = nn.ModuleList([SubModel(num_const, 
                                                  num_var, 
-                                                 num_hidden, 
-                                                 num_out, 
+                                                 num_hidden,
+                                                 num_out,
                                                  num_groups,
-                                                 independent)
-                                        for num_groups in self.num_groups])
+                                                independent)
+                                        for num_groups in self.num_groups]
 
+        self.device = device
+                                       
     def reset_hidden(self):
         for submodel in submodels:
-            submodel.reset_hidden()
+            submodel.reset_hidden()     
 
-    def forward(self, items, day):
+    def forward(self, day, items):
+        day = day.to(self.device)
```

Comment:
at this point an error occurs:
  File "/Users/mauriceverbrugge/github/MLiP-2020/model.py", line 55, in forward
    for i, items in enumerate(items.split(self.num_groups, dim=-1)):
  File "/usr/local/lib/python3.7/site-packages/torch/tensor.py", line 386, in split
    return super(Tensor, self).split_with_sizes(split_size, dim)